src/pages/workspaces/List.js

```diff
@@ -137,6 +137,40 @@ const NewWorkspaceCard = pure(({ listView, onClick }) => {
   ])
 })
 
+export const LargeFadeBox = ({
+  padding = '1.5rem',
+  backgroundColor = colors.gray[5],
+  borderColor = colors.gray[3],
+  fadePoint = '60%',
+  children
+}) => {
+  return div({
+    style: {
+      background: `linear-gradient(to bottom, white 0%, ${colors.gray[5]} 4%`,
+      borderRadius: '8px 8px 0 0',
+      margin: '1.5rem'
+    }
+  }, [
+    div({
+      style: {
+        height: '.25rem',
+        border: `1px solid ${colors.gray[3]}`,
+        borderBottom: 'none',
+        borderRadius: '8px 8px 0 0'
+      }
+    }),
+    div({
+      style: {
+        borderWidth: 1,
+        borderStyle: 'solid',
+        borderImage: `linear-gradient(to bottom, ${colors.gray[3]}, ${colors.gray[5]} 4%) 1 100%`,
```

Comment:
TOL: Using a percentage for the color stop means that the fade length depends on the size of the box, e.g. the number of workspaces in this case. Is that what we want, design-wise, or should we use a fixed length? Or something fancy with `calc()`?

Reply:
I'm not really sure--perhaps a fixed length so we don't end up with too much white at the top just because the workspace list is long?